lms/resources/lti_launch.py

```diff
@@ -178,12 +173,7 @@ def canvas_sections_supported(self):
             # Canvas course sections feature was released.
             return False
 
-        try:
-            application_instance = self._application_instance_service.get_current()
-        except ApplicationInstanceNotFound:
-            return False
-
-        return bool(application_instance.developer_key)
+        return bool(self.js_config.application_instance.developer_key)
```

Comment:
Does this explode with an attribute error now if `application_instance` is None? How is this handled?